spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/main/java/org/springframework/boot/configurationprocessor/PropertyDescriptorResolver.java

```diff
@@ -125,6 +138,14 @@ private ExecutableElement findMatchingGetter(TypeElementMembers members, List<Ex
 		return candidates.get(0);
 	}
 
+	private ExecutableElement findMatchingSetter(TypeElementMembers members, List<ExecutableElement> candidates,
```

Comment:
this is a a copy of findMatchingGetter, maybe a generalization for getters and setters could be introduced?